src/rfac/rfrmin.c

```
@@ -0,0 +1,458 @@
+#define SHIFT_DE 1
+#endif
+
+/*!
+ * Returns the energy range of the IV curve.
+ *
+ * \param iv Pointer to #rfac_iv containing energy data
+ * \return Energy range of IV curve.
+ * \note Returns zero if either \p iv or \c data member are \c NULL
+ * (or \c n_eng data member is 0).
+ */
+real rfac_iv_get_energy_range(const rfac_iv *iv) {
```

Cppcheck (2.19.0)

[style] The function 'rfac_iv_get_energy_range' should have static linkage since it is not used outside of its translation unit.

(staticFunction)